src/System.IO.FileSystem/src/System/IO/Directory.cs

```diff
@@ -232,12 +232,6 @@ public static string GetDirectoryRoot(string path)
             return root;
         }
 
-        internal static string InternalGetDirectoryRoot(string path)
```

Comment:
Moved to a new file called FileSystem.cs. See full explanation of this change in the comment I left in that file.